### ISSUE TITLE
GitHub release attributed to wrong person

### ISSUE THREAD

**Issue by glikely** [travis-ci/travis-ci#8953]
It seems like GitHub releases created by Travis-CI are always attributed to the same person, regardless of who cut the release. I cannot find a way to get the release to be attributed to the person who signed the release tag. For example, on the devicetree-specification repo, the latest release is v0.2-rc1, which is tagged with a signed tag by me (grant.likely@arm.com). But the release always gets attributed to Rob Herring because we're using his GitHub OAuth key for Travis-ci integration. Here's the link:

https://github.com/devicetree-org/devicetree-specification/releases

The release says Rob Herring, but if you click on the [verified] box, it shows my name.

Sometimes I cut the release, sometimes Rob does. Is there any way to get Travis-CI to create GitHub releases in the name of the person who tagged the release?

**Comment by glikely** [comment]
Right now I'm working around the problem by manually changing the OAuth key in .travis.yml and committing that to the repo when I cut a release, but there must be a better way.

**Comment by BanzaiMan** [comment]
I do not think this is easily achievable, as git commits that you push to GitHub are not tied to GitHub users who create the GitHub Releases.

In order to achieve what you are asking, we will need to have (or means to retrieve) all the GitHub API tokens that can potentially create GitHub Releases in all the repositories.
